impeller/compiler/switches.cc

```diff
@@ -127,11 +127,17 @@ Switches::Switches(const fml::CommandLine& command_line)
 
     // fml::OpenDirectoryReadOnly for Windows doesn't handle relative paths
     // beginning with `../` well, so we build an absolute path.
+
+    // Get the current working directory as a utf8 encoded string.
```

Comment:
Can this be unit tested in compiler_unittests.cc?

Reply:
Done.